---
title: Meeting minutes from the Biology Postdoc Cohort at Emory 
author: |
 | Matthew Malishev^1^* & Molly Gallagher^1^   
 |  
 | _^1^ Department of Biology, Emory University, 1510 Clifton Road NE, Atlanta, GA, USA, 30322_
#bibliography:/Users/malishev/Documents/Melbourne Uni/Thesis_2016/library.bib
fontsize: 10
geometry: margin=1in
documentclass: article
linkcolor: blue
urlcolor: blue
citecolor: red
output:
  html_document:
    highlight: tango
    code_folding: hide
    toc: yes
    toc_depth: 2
    number_sections: no
    toc_float: yes
  pdf_document:
    includes:
      in_header: # add .tex file with header content
    highlight: tango
    template: null
    toc: yes
    toc_depth: 2
    number_sections: false
    fig_width: 4
    fig_height: 5
    fig_caption: true
    df_print: tibble 
    citation_package: biblatex # natbib
    latex_engine: xelatex #pdflatex # lualatex
    keep_tex: true # keep .tex file in dir 
  word_document:
    highlight: tango
    keep_md: yes
    pandoc_args: --smart
    #reference: mystyles.docx
    toc: yes
inludes:
  before_body: before_body.tex
subtitle: 
tags:
- nothing
- nothingness
params: 
  dir: "/Users/malishev/Documents/Emory/admin/postdocs/emory_postdocs/minutes"
  date: !r Sys.Date()
  session: !r sessionInfo()  
  version: !r getRversion()
  email: "matthew.malishev@emory.edu"
  doi: https://github.com/darwinanddavis/emory_postdocs
classoption: portrait
# ^['https://github.com/darwinanddavis/UsefulCode'] # footnote
vignette: >
  %\VignetteIndexEntry{Useful R code}
  %\VignetteEncoding{UTF-8}
  %\VignetteEngine{knitr::rmarkdown}
---

<script type="text/x-mathjax-config">
  MathJax.Hub.Config({ TeX: { equationNumbers: {autoNumber: "all"} } });
</script>

```{r echo = FALSE}
# library(rmarkdown)
# setwd("")
# f <- list.files()[1]
# render(f, output_format='pdf_document')
```

```{r, set-options, echo = FALSE, cache = FALSE}
options(width=100)
knitr::opts_chunk$set(
 eval = TRUE, # run all code
 # echo = FALSE, # show code chunks in output 
 comment = "",
 tidy.opts=list(width.cutoff=100), # set width of code chunks in output
 tidy=TRUE, # make output as tidy
 message = FALSE,  # mask all messages
 warning = FALSE, # mask all warnings 
 size="small" # set code chunk size
)

# https://github.com/ucb-stat133/stat133-fall-2016/blob/master/hws/hw02-tables-ggplot.Rmd
knitr::opts_knit$set(root.dir=paste0(params$dir,"/")) # set working dir

setwd(paste0(params$dir,"/")) # for running just in R not knitr
```

\newpage   

Date: `r params$date`  
R version: `r params$version`  
*Corresponding author: `r params$email`  
This document can be found at `r params$doi`  

\  

Session info   

```{r echo=F, echo=F}
params$session
```      

\newpage  

## Overview

This document contains the meeting minutes from the Biology Postdoc Cohort at Emory.  

The group hosts regular meetups to learn about what the postdocs in Biology at Emory are doing, harness cool research skills and tools that everyone uses, foster research overlaps, brainstorm and troubleshoot ideas, discuss weird results that nobody knows the answer to, practise upcoming seminars, and simply build a stronger postdoc culture in Biology at Emory.    

Questions and suggestions welcome at `r params$email`.    

\newpage  

## TO DO list  

- Add attending postdocs to OPE Emory list (Beverly)    
* Come up with a cooler name for the group  
* Workshop ideas  
* Writing retreat  
* Aim for a collaborative paper   

### _Next meetup:_ March 1, 2019    

\newpage    
  
## March 1, 2019  

Meetup on CRISPR, presented by Aileen Berasategui from the Gerardo Lab.      

### "CRISPR, or how a mysterious DNA sequence turned into the most important discovery of this century"  

We had some stimulating conversation over the usefulness of CRISPR, its future, its contribution to modernising science, and the many ethical challenges not just academics, but also the public will face.    

Aileen fielded the questions raised at the end of the presentation and has provided some more in-depth answers below.    

**Are the viral DNA pieces integrated in the CRISPR region random?**  

My instinct was to say yes, but then I have realized that certainly not. They all have a so called PAM sequence (Protospacer Adjacent Motif) at the beginning and this cannot possibly be random. For CRISPR type II, this sequence is NGG (any nucleotide, guanine, guanine). So I have read further and it has been demonstrated that the PAM sequence is essential for Cas9 recognition. (Hille and Charpentier 2016).   
 
**Have viruses evolved counteradaptations to CRISPR?**  

YES! I thought so but I had no specific examples. They can avoid recognition by Cas proteins by mutating their PAM sequence (very neat).  
 
**Do all spacers have the same size?**  

CRISPR spacers can range between 21 and 72 nucleotides long (they normally are between 32 and 38 nucleotides). Something I did not mention yesterday is that a bacterial (or achaeal) cell can have more than one CRISPR locus. The length of the spacers remain constant within one locus, but can vary between locus within the same genome, and of course between cells.    
 
**Where does genome editing by CRISPR take place?**  

In bacteria and archaea it takes place in the cytosol (no nucleus present). In eukaryotes, the editing happens in the nucleus. But the complete mechanism depends on how you deliver the CRISPR-Cas complex to cells. In any case, translation of the Cas9 RNA to protein has to happen in the cytosol where it will also binds to the guideRNA. Together they go back to the nucleus to edit their target host DNA.  
 

**Can CRISPR be horizontally transmitted?**  

There is evidence suggesting as much. However, I have dug a little bit deeper and I have found that actually (and it makes total sense), CRISPR tends to prevent HGT. That is because CRISPR recognizes foreign DNA floating in the cytosol and at some point the genes being transferred will be floating around before being integrated into the genome. So, perhaps the answer is yes, but rarely? Apparently, it depends on the method for HGT (transduction vs. transformation vs. conjugation). (Watson et al. 2018, mBio).    

\newpage  

## February 8, 2019   

In this meeting we created our own research impact statements to practice distilling our specialised research areas into lay terms.  

The general recipe to follow:  
- What you did  
- Who was helped  
- In what way are they better off  
(different people have different perspectives on importance of the order of these points)  

**Molly Gallagher**  
Our models show that for viral infections, treating patients with defective viral particles that interfere with normal viral replication can reduce the severity of symptoms, and may reduce the chance of transmitting the infection to others.  

**Matt Malishev**  
I’m interested in how diseases spread in nature. I investigate how environmental change shapes the energetics of parasite populations transmitting schistosomiasis. I apply metabolic theory to simulate human infection probability and exposure risk in space and time to inform useful biocontrol strategies.    

**Rohan Mehta**      
I develop new theoretical tools to help biologists study how the distribution and movement of populations affects their ability to adapt to environments that vary in space.  

**Lewis Bartlett**      
I study how different beekeeping practices affect the ways bee diseases spread and how deadly they become. I use laboratory studies, field tests, computer simulations, and maths to predict and test which actions beekeepers should take to prevent infectious diseases.  

**Scott Villa**  
My research focuses on understanding how and why there are so many different species. I am interested in how natural selection influences traits critical for mating. Specifically, I experimentally evolve parasites on new hosts to explore how new species form under varying environmental, genetic, and demographic scenarios.  

**Laramie Lemon**    
I use yeast to investigate how genes are activated. I focus on how the physical structure of DNA affects the ways it’s read, copied, and translated to produce proteins by different mechanisms.  

**Venkat Talla**  
I study how populations become different species by measuring the genetic differences and how their genomes change with time. I use genomic data of Monarch butterflies to answer questions about species divergence, natural selection and conservation.  

\newpage  

Some resources on science communication.  

![Language matters](/Users/malishev/Documents/Emory/admin/postdocs/emory_postdocs/resources/sci_comm_GOS.jpg)      

\newpage    
  
## January 11, 2019   

IMPACT statement workshop  
- Come up with a Impact Statment (3 line summary of your research) for next meeting.    

  
[XKCD](https://xkcd.com/) inspired page for communicating your ideas using just the ten hundred most commonly-used words: [The Up-Goer Five Text Editor](http://splasho.com/upgoer5/)  
  
      
![Check out this page for a practical way of communicating your super complex research: [The Up-Goer Five Text Editor](http://splasho.com/upgoer5/)](upgoer5.jpeg)    


### Some writing guides from the masses  

[The Elements of Style by Strunk and White](https://www.amazon.com/Elements-Style-William-Strunk-Jr/dp/194564401X)  

[Revising Prose by Richard Lanham](http://people.vetmed.wsu.edu/jmgay/courses/documents/TheParamedicMethod.pdf)  

[Idead into Words by Elise Hancock](https://www.amazon.com/Ideas-into-Words-Mastering-Science/dp/0801873304/ref=pd_sim_b_1)  **Matt's personal favourite**    
  

\newpage    

## December 14, 2018  

## _Biosketch of postdocs_  

**Matt Malishev**  
Civitello lab  
Bioenergetics and individual-based modelling of host-parasite dynamics of human schistosome populations; spatial simulation modelling; metabolic theory  

**Molly Gallagher**  
Koelle lab  
Disease ecology; differential equation models; current work focuses on modeling defective interfering particles in influenza

**Aileen Berasategui Lopez**  
Gerardo lab  
The genomic and chemical basis of host-fidelity  

**Caitlin Conn**  
Gerardo lab  
Host range and its genetic basis in a mycoparasite  

**Jeremy Harris**  
Koelle lab   
Current IAV modeling: Passage study modeling, estimating bottleneck sizes  

**Mary Bushman**  
Rustom lab  
Linking within- and between-host dynamics of infectious diseases (modeling)  

**David Nicholson**  
Prinz lab  
"Lifelong Learning Machines" -- continual machine learning algorithms  

**Julien Catanese**    
Jaeger lab      


**Derrick Morton**      
Corbett lab    
Studying how defects in RNA processing lead to neurodegenerative disesase    

**Scott Villa**  
Gerardo lab    
Role of endysymbionts in driving host reproductive isolation and adaptive radiation  
  
**Rohan Mehta**  
Weissman Lab  
Evolution in spatially-structured populations  
  
### Workshop ideas  
IMPACT workshop  
- Impact of science + advancement in research    
Contact: Derrick   
  
\newpage  

## November 30, 2018  

### Outcomes for postdoc group  
Meetings every second and fourth week    

Talk therapy among working class postdocs  

Intro talks from postdocs to group 

* Lightning, 2-min talk for group  

* New postdocs give a brief intro talk for their first meeting     

Postdoc mentorship group for assisting grad students during qualifying exams  

* Online directory of postdocs showcasing background and expertise. Include non-science background stuff, such as applying for international universities.    
The journal peer review system and open access science  

* Using _bioarXiv_ and pre-prints    
* Open access science  
* Writing a paper on the science peer review system from postdoc perspective   

\newpage  

## November 1, 2018  

#### Ideas for things people want

Collaborate on overlapping research  
Brainstorm ideas  
Present new results  
Practise conference talks  
Writing retreats  
Regular coding/math club  

#### Bigger ideas  

Combine other labs/departments    
- other biology floor levels  
- math/env sciences  
  
\newpage  

## Links and ideas   








